resources/puppetlabs/lein-ezbake/template/global/controller.sh.erb

```diff
@@ -13,7 +15,9 @@ build_os=$1
 build_ver=$2
 
 BASEPATH='pkg_artifacts'
-mkdir $BASEPATH
+if [ ! -d "$BASEPATH" ]; then
+  mkdir $BASEPATH
+fi
```

Comment:
Why not `mkdir -p`?

Reply:
This felt more explicit for what I was trying to do. I've mostly used `mkdir -p` when making a directory down a few levels, so running it when making a directory in PWD feels strange :woman_shrugging:

Reply:
hmm. I see what you mean, though I can't claim to know what's idiomatic. I'm just generally a fan of shorter/cleaner. 
also, theoretically, couldn't `$BASEPATH` have several levels?

Reply:
`BASEPATH` is hard-coded one line above this, so it will always have one level.

Reply:
oh yeah. It could theoretically change though..? idk, I don't have strong feelings. ¯\_(ツ)_/¯

Reply:
if we change it we can change how the directory is made though, since we'll be updating this file anyways. In bash I tend to prefer verbosity if it makes it more clear/explicit, which I think this does.

Reply:
fair enough 😊